WebRoot/css/zm.css

```diff
@@ -4021,23 +4021,69 @@ HTML>BODY .appt-selected .appt_allday_body
 }
 
 /* ZmChooseFolderDialog and ZmPickTagDialog */
+
+#ChooseFolderDialog {
+	@ChooseFolderDialog@
+}
+
+#ChooseFolderDialog .WindowOuterContainer {
+	@ChooseFolderDialogContainer@
+}
+
 .ChooserDialog {
-	width:320px;
+	@ChooseFolderDialogBody@
 }
 
 .ChooserDialog .DwtInputField {
-	padding-bottom:3px;
-	padding-top:3px;
+	@ChooseFolderDialogInputContainer@
 }
 
-.ChooserDialog INPUT {
-	width:100%;
+#ChooseFolderDialog .ChooserDialog INPUT {
+	@ChooseFolderDialogInputField@
 }
 
 .ChooserDialog .overview {
-	background-color:white;
-	border:1px solid black;
-	overflow:auto;
+	@ChooseFolderDialogFolderContainer@
+}
+
+.ChooserDialog .dialogOverview {
+	@ChooseFolderDialogFolderContainerContent@
+}
+
+.ChooserDialog .DwtTreeItem {
+	@ChooseFolderDialogFolderItem@
+}
+
+.ChooserDialog .DwtTreeItemLevel1ChildDiv {
+	padding-left:0px;
```

Comment:
Would it be better to have some property for this? I guess this is just to remove breathing space. If you feel no property is needed here,how about moving to `skin.css`